client/package.json

```diff
@@ -14,10 +14,11 @@
     "babel-plugin-transform-decorators": "^6.24.1",
     "core-js": "^3.3.2",
     "lodash": "^4.17.20",
+    "@mdi/font": "^4.5.95",
     "material-design-icons-iconfont": "^5.0.1",
     "moment": "^2.27.0",
     "remove-accents": "^0.4.2",
-    "sbc-common-components": "2.3.3",
+    "sbc-common-components": "^2.4.3",
```

Comment:
Please remove the caret. Since we generally don't know what changes in sbc-common-components, it's better to control what version we are using and then update manually.

Reply:
ah right, will do, thanks!

Reply:
Updated in next commit 👍